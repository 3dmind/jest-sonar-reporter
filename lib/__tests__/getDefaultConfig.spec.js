'use strict';

const getDefaultConfig = require('../getDefaultConfig');

describe('getDefaultConfig', () => {
  test('should retrieve default configuration', () => {
    expect.assertions(1);

    const root = '/';

    const defaultConfig = getDefaultConfig(root);

    expect(defaultConfig).toMatchSnapshot();
  });
});
