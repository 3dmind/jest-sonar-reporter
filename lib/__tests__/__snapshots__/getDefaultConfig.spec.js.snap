// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getDefaultConfig should retrieve default configuration 1`] = `
Object {
  "indent": 2,
  "reportFile": "test-report.xml",
  "reportPath": "/",
  "sonar56x": false,
  "useRelativePath": false,
}
`;
